tests/vmi_multus_test.go

```diff
@@ -52,23 +52,11 @@ var _ = Describe("Multus Networking", func() {
 	virtClient, err := kubecli.GetKubevirtClient()
 	tests.PanicOnError(err)
 
-	nodes, err := virtClient.CoreV1().Nodes().List(v13.ListOptions{})
+	computeNodes, err := virtClient.CoreV1().Nodes().List(v13.ListOptions{
+		LabelSelector: "node-role.kubernetes.io/compute=true",
```

Comment:
I think this will not work for one node.

```yaml
apiVersion: v1
items:
- apiVersion: v1
  kind: Node
  metadata:
    annotations:
      flannel.alpha.coreos.com/backend-data: '{"VtepMAC":"76:2c:b2:54:86:78"}'
      flannel.alpha.coreos.com/backend-type: vxlan
      flannel.alpha.coreos.com/kube-subnet-manager: "true"
      flannel.alpha.coreos.com/public-ip: 192.168.66.101
      kubeadm.alpha.kubernetes.io/cri-socket: /var/run/dockershim.sock
      node.alpha.kubernetes.io/ttl: "0"
      volumes.kubernetes.io/controller-managed-attach-detach: "true"
    creationTimestamp: "2019-02-06T11:56:10Z"
    labels:
      beta.kubernetes.io/arch: amd64
      beta.kubernetes.io/os: linux
      kubernetes.io/hostname: node01
      node-role.kubernetes.io/master: ""
    name: node01
```

Reply:
@SchSeba If that one node is a compute node, shouldn't it have 
`node-role.kubernetes.io/compute: "true"`
If not - how do you recommend to resolve this?

Reply:
can you try to use "node01" as the node name just like other test use PVC.

after the tests run with node01 lets see what the CI have to said.

Reply:
Resolved